hexkit/providers/akafka.py

```diff
@@ -15,23 +15,42 @@
 #
 
 """
-Apache Kafka specific event publishing provider.
+Apache Kafka-specific event publisher and subscriber provider.
+They correspond to the `EventPublisherProtocol` and `EventSubscriberProtocol`, respectively.
 
 Require dependencies of the `akafka` extra. See the `setup.cfg`.
 """
 
 import json
+import logging
 
-from kafka import KafkaProducer
+from kafka import KafkaConsumer, KafkaProducer
+from kafka.consumer.fetcher import ConsumerRecord
 
+from hexkit.base import InboundProviderBase
 from hexkit.custom_types import JsonObject
 from hexkit.protocols.eventpub import EventPublisherProtocol
+from hexkit.protocols.eventsub import EventSubscriberProtocol
 
 
 class NonAsciiStrError(RuntimeError):
     """Thrown when non-ASCII string was unexpectedly provided"""
 
-    pass  # pylint: disable=unnecessary-pass
+    pass
```

Comment:
`pass` is actually not needed when there is a docstring.

Reply:
Oh interesting, didn't know that. So pylint was right. Thank you.